internal/cypress/config.go

```diff
@@ -98,6 +100,13 @@ func FromFile(cfgPath string) (Project, error) {
 	}
 	p.Cypress.ProjectPath = cPath
 
+	// Check rootDir if it is set.
+	if p.RootDir != "" {
+		if _, err := os.Stat(p.RootDir); err != nil {
+			return p, fmt.Errorf("unable to locate the rootDir folder in %s", p.RootDir)
```

Comment:
```suggestion
			return p, fmt.Errorf("unable to locate the rootDir folder %s", p.RootDir)
```

Reply:
Slight suggestion since you're looking for folder `p.RootDir` and not in `p.RootDir`.

Reply:
I fixed it.